conduct.md

```diff
@@ -220,6 +222,6 @@ Any decision made by the instructor for the given infraction(s) will be final.
 
 ## Names of Individuals who Agree to Adhere to the Guide
 
-:heavy_exclamation_mark: TODO: Add your name
-Janyl Jumadinova, Bennett Westfall, Zackery Devers, Natnael Kahesay, Kobe Coleman, Adriana Solis, Favour Ojo, Kai'lani Woodward, Zoe White
+Janyl Jumadinova, Bennett Westfall, Zackery Devers, Natnael Kahesay, Kobe Coleman, Adriana Solis, Favour Ojo, Kai'lani Woodward, Zoe WhiteJanyl Jumadinova, Lex Caldwell, Nathan Loria, Kaung Myat Htet(KM), Linh Nguyen Mai
```

Comment:
JJ's name appears twice, so if you could just remove one of them that would be great :)